libcxx/test/std/language.support/support.exception/propagation/exception_ptr_move_assignment.cpp

```diff
@@ -0,0 +1,44 @@
+//===----------------------------------------------------------------------===//
+//
+// Part of the LLVM Project, under the Apache License v2.0 with LLVM Exceptions.
+// See https://llvm.org/LICENSE.txt for license information.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+//
+//===----------------------------------------------------------------------===//
+
+// <exception>
+
+// typedef unspecified exception_ptr;
+
+// Test the move assignment of exception_ptr
+
+#include <exception>
+#include <utility>
+#include <cassert>
+
+#include "test_macros.h"
+
+int main(int, char**)
```

Comment:
Please format the tests.

Reply:
Fixed.

But I still find it annoying that libc++ mixes styles. I copied the style from the existing `exception_ptr.pass.cpp`, and still that decision was apparently the wrong one.

I made it a habit to always follow the code style of surrounding code. Usually that consistency with existing code is appreciated by open-source projects. Am I the only contributor who runs into that? Can we reformat all code to follow the same code style?